book/src/builders.md

```diff
@@ -157,20 +157,64 @@ By default, Lighthouse is strict with these conditions, but we encourage users t
 - `--builder-fallback-disable-checks` - This flag disables all checks related to chain health. This means the builder
   API will always be used for payload construction, regardless of recent chain conditions.
 
-## Builder Profit Threshold 
+## Builder Profit Threshold
 
 If you are generally uneasy with the risks associated with outsourced payload production (liveness/censorship) but would
 consider using it for the chance of out-sized rewards, this flag may be useful:
 
 `--builder-profit-threshold <WEI_VALUE>`
 
-The number provided indicates the minimum reward that an external payload must provide the proposer for it to be considered 
+The number provided indicates the minimum reward that an external payload must provide the proposer for it to be considered
 for inclusion in a proposal. For example, if you'd only like to use an external payload for a reward of >= 0.25 ETH, you
-would provide your beacon node with `--builder-profit-threshold 250000000000000000`. If it's your turn to propose and the 
+would provide your beacon node with `--builder-profit-threshold 250000000000000000`. If it's your turn to propose and the
 most valuable payload offered by builders is only 0.1 ETH, the local execution engine's payload will be used. Currently,
-this threshold just looks at the value of the external payload. No comparison to the local payload is made, although 
+this threshold just looks at the value of the external payload. No comparison to the local payload is made, although
 this feature will likely be added in the future.
 
+## Checking your builder config
+
+You can check that your builder is configured correctly by looking for these log messages.
+
+On start-up, the beacon node will log if a builder is configured:
+
+```
+INFO Connected to external block builder
+```
+
+At regular intervals the validator client will log that it successfully registered its validators
+with the builder network:
+
+```
+INFO Published validator registrations to the builder network
```

Comment:
Do you want to have a list of all of the validators that were registered?

Reply:
We do include a count of validators registered in this log. Do you think that seems adequate? We could include indices, but if people are running with hundreds of validators this log might become a bit overwhelming because it runs every epoch and should usually be unchanged.

Reply:
Completely legit point. I agree it is sufficient with the quantity especially if registered each epoch .

Reply:
I decide to leave out the parameters in the examples because they take up a bunch of space (and are liable to minor changes). Users will see them while checking their actual logs